language/golang/build-images.md

```diff
@@ -427,9 +427,7 @@ content:
 ```dockerfile
 # syntax=docker/dockerfile:1
 
-##
 ## Build
-##
```

Comment:
heh. funny; it looks like it picked these up as Markdown, not as Dockerfile comments? (is that a bug in the linter?)

Reply:
Yeah this is a bug as far as I can tell :cry:

I'll try and get a reproducible example at some point, and submit a patch upstream to fix when I have the time. It's strange, since this is inside a code block :confused: I think what's happened is it's picked up as a weird type of header? And then gone "this header isn't like any of the others", and thrown an error.

Reply:
😄 ah, yes, fun one indeed.

Changing it looks fine though for this (doesn't change the intent, so should be fine to change and make the linter happy)